rules/manager.go

```diff
@@ -37,9 +37,9 @@ import (
 const (
 	namespace = "prometheus"
 
-	ruleTypeLabel     = "rule_type"
-	alertingRuleType  = "alerting"
-	recordingRuleType = "recording"
+	RuleTypeLabel     = "rule_type"
```

Comment:
IMO the other places where we currently export things that aren't used from the outside should also be un-exported.